mitodl/open-discussions
# Add title and headline fields to edit channel appearance page

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1094 

#### What's this PR do?
Adds a title and replaces the description field with the headline field, which is `public_description` on the backend API

#### How should this be manually tested?
Edit the title and channel headline and verify that the changes are persisted. You should also not be able to put more than 80 characters in the headline and you should see a validation error when the title is missing.
